template-downtime.yaml

```diff
@@ -0,0 +1,35 @@
+# Class describes whether or not the downtime was planned. Choose one of the
+# following classes:
+# - SCHEDULED: donwtime that has been scheduled long before the actual downtime
+# - UNSCHEDULED: donwtime that has occurred due to an event such as a sudden
+#   hardware failure, or any unforseen circumstances
+- Class: <TEXT>
+  # Summary of the reason for the downtime
+  Description: <TEXT>
+  # Downtime severity in descending order:
+  # - Outage
+  # - Severe
+  # - Intermittent Outage
+  # - No Significant Outage Expected
+  Severity: <TEXT>
+  # Start date and time of the donwtime in the following format:
+  # <MONTH> <DAY>, <YEAR> <UTC TIME>
+  # e.g. Mar 28, 2017 22:00 PM UTC
```

Comment:
Can you remove the AM/PM? It's already in 24-hour time. I know the existing data contains it, but I'd like the new data to suck less.

Also, it doesn't have to be in UTC time but they may need to provide the time zone so say something like
```
# <MONTH> <DAY>, <YEAR> <TIME> <TIME ZONE>
# e.g. Mar 28, 2017 17:00 CDT
```